samples/notebooks/quickstart.ipynb

```diff
@@ -0,0 +1,572 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def list_blobs(bucket_name: str) -> str:\n",
+    "  '''Lists all the blobs in the bucket.'''\n",
+    "  import subprocess\n",
+    "\n",
+    "  subprocess.call(['pip', 'install', '--upgrade', 'google-cloud-storage'])\n",
+    "  from google.cloud import storage\n",
+    "  storage_client = storage.Client()\n",
+    "  bucket = storage_client.get_bucket(bucket_name)\n",
+    "  list_blobs_response = bucket.list_blobs()\n",
+    "  blobs = ','.join([blob.name for blob in list_blobs_response])\n",
+    "  print(blobs)\n",
+    "  return blobs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.2 Create a lightweight Python component"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import kfp.components as comp\n",
+    "\n",
+    "# Converts the function to a lightweight Python component.\n",
+    "list_blobs_op = comp.func_to_container_op(list_blobs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.3 Define pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import kfp.dsl as dsl\n",
+    "\n",
+    "# Defines the pipeline.\n",
+    "@dsl.pipeline(name='List GCS blobs', description='Lists GCS blobs.')\n",
+    "def pipeline_func(bucket_name=dsl.PipelineParam('bucket')):\n",
+    "  list_blobs_task = list_blobs_op(bucket_name)\n",
+    "\n",
+    "# Compile the pipeline to a file.\n",
+    "import kfp.compiler as compiler\n",
+    "compiler.Compiler().compile(pipeline_func, 'list_blobs.pipeline.tar.gz')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Build a new Docker container image from a Python function.\n",
+    "Refer [here](https://runnable.com/docker/python/dockerize-your-python-application#alternatives) for more information."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Wrap an existing Docker container image using `ContainerOp`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.1 Create a Docker container\n",
+    "Create your own container image that includes your program. If your component creates some outputs to be fed as inputs to the downstream components, each separate output must be written as a string to a separate local text file by the container image. For example, if a trainer component needs to output the trained model path, it can write the path to a local file `/output.txt`. The string written to an output file cannot be too big. If it is too big (> 500kb), you can save the output to an external persistent storage and pass the storage path to the next component.\n",
```

Comment:
How did you arrive at the 500kb limit?

Reply:
I think someone from Kubeflow Pipelines team shared this limit with me. How about changing it to >> 100 kB? I can also delete it.